token-metadata/program/src/processor/mod.rs

```diff
@@ -435,3 +450,24 @@ fn has_programmable_metadata<'a>(
 
     Ok(false)
 }
+
+/// Checks if the instruction's accounts contain a pNFT metadata.
+///
+/// We need to determine if we are dealing with a pNFT metadata or not
```

Comment:
These comments need slight updating to reference checking for Token Record instead of Metadata account.

Reply:
Done.